artemis-journal/src/main/java/org/apache/activemq/artemis/core/io/mapped/MappedFile.java

```diff
@@ -301,23 +206,24 @@ public void zeros(int position, final int count) throws IOException {
       if (toZeros > 0) {
          PlatformDependent.setMemory(start, toZeros, (byte) 0);
       }
-
+      //do not move position!
```

Comment:
I don't understand this comment... you are moving position and writing do not move?

Reply:
Nope, `this.position` isn't moved, only the local variable named `position` is: the correct comment could be `//do not move this.position`